lnwire/funding_created.go

```diff
@@ -26,6 +27,13 @@ type FundingCreated struct {
 	// transaction.
 	CommitSig Sig
 
+	// PartialSig is used to transmit a musig2 extended partial signature
+	// that also carries along the public nonce of the signer.
+	//
+	// NOTE: This field is only populated if a musig2 taproot channel is
+	// being signed for. In this case, the above Sig type MUST be blank.
+	PartialSig *PartialSigWithNonce
```

Comment:
ok, I see now that the Signing nonce that was originally sent in open-channel (as described in the bolt proposal) is now sent here? 

just out of interest, what was the reason for this change? cause from my understanding, If we sent sent this nonce in open-channel then here we could reuse the `signature` field to send the partial sig (with some padding)

Reply:
ah - Is the reason that there are fewer bytes to send across on connection reestablish?

Reply:
The general idea is that we can defer sending the nonce we use to sign to the very last step. This lets the signer mix in more randomness, and also means they don't need to commit to session creation until there's a state to create/accept. 

It also makes chan reest simpler as well, since you're already generating the nonce at signing time, so you can either retransmit what you had, or just generate a new nonces anew.